src/projects/machinacommons_adaptor-ex.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: layouts/project.liquid
+tags: ["Kultur","Theater","Games"]
+id: adaptorex
```

Comment:
If you use `adaptorex` as the ID, your thumb image has to be `adaptorex_thumb.png` without "-" as it is currently.